### PR TITLE
Kaptain 1.2.0-1.0.0 GA Release

### PR DESCRIPTION
**Pull request by akirillov**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-74180

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by ianhellstrom — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: Version Support Policy
+menuWeight: 5
+excerpt: Kaptain's supported version policy
+beta: false
+enterprise: false
+---
+
+D2iQ supports `N-2` of the latest `MAJOR.MINOR` release of Kaptain.
+With the GA release of Kaptain 1.0.0, support continues for Kaptain 1.0, 0.6, and 0.5, including all patch releases.
+Support for Kaptain 0.4.0 expires.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md:13]:
```suggestion
With the GA release of Kaptain 1.0.0, support continues for Kaptain 1.0, 0.6, and 0.5, including all patch releases.  Support for Kaptain 0.4.0 expires.
```

File: pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Users should upgrade their Kaptain installations clusters with every new release to stay up to date with the latest features and bug fixes.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md:15]: My understanding is that Kaptain does not currently support in place 'upgrades'.  I.e. you must uninstall and reinstall Kaptain if you wish to use a newer version.    

If so, then I don't think we should use the word 'upgrade' here.  Instead, I'd suggest:

```suggestion
D2iQ recommends that you install the newest Kaptain version available to stay up to date with the latest features and bug fixes.
```

File: pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Supported Konvoy Versions
+
+Kaptain 1.0.0 runs on Konvoy 1.7.
+
+## Supported Kommander Versions
+
+Kaptain 1.0.0 is not yet supported on Kommander.
+
+Note that it is possible to use [Kommander's Dex instance](../custom-configuration/external-dex) for authenticating users in Kaptain.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md:25]: These two sentences seem to contradict each other.  Please clarify what we mean by 'not yet supported'?   Is that you cannot install Kaptain using Kommander?  Or something else?

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md (hunk 3 above)
Comment (ianhellstrom) [reply pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md:25]: You cannot install Kaptain from Kommander but you can use Kommander's Dex instance for authentication.

**Pull request merged**